checklists/adding-fonts-to-the-wsk.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Adding Fonts to the WSK
+category: Front-End
+date: "2020-03-20"
+tags: ['font', 'scss']
+description: Here is the checklist that we follow to import fonts to our projects.
+---
+
+- [ ] Get the fonts!  
+Search through the web but if you can't manage to find kindly ask your designer to provide you.
+
+- [ ] Convert your fonts  
+We use fonts as 'woff', 'woff2' and 'ttf' formats in our projects so you need to convert your fonts into those. You can use 'fontplop' desktop app for it.
```

Comment:
I don't know how you guys are proceeding if the font is in .otf format?

Reply:
Fontplop accepts both ttf and otf formats.